config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "Kevin",
```

Comment:
get this from process.ENV.DB_USERNAME because your teammate has a different username in his own local computer: https://github.com/izelnakri/nycda-blog-application/commit/e929a158f3f3f3d20f63b853ad0af91986f9772f

you need to add DB_USERNAME environment variable to your ~/.bash_profile